lib/flex_commerce_api/json_api_client_extension/builder.rb

```diff
@@ -5,8 +5,9 @@ def initialize(*)
         super
         @temp_search_criteria = nil
       end
-      def temp_search(options = {})
+      def temp_search(options = {}, available_to_browse_only=true)
```

Comment:
This means that the options will need to be surrounded with {} to allow the extra flag to be set - a bit ugly but if 100% needed then go for it.